package.json

```diff
@@ -2,25 +2,25 @@
 	"name": "data-update-github",
 	"version": "0.1.0",
 	"description": "A remote data change triggers the issue of a pull request to a specified Github repo.",
+	"main": "index.js",
 	"scripts": {
 		"lint": "grunt lint",
 		"test": "node tests/all-tests.js"
 	},
 	"author": "Inclusive Design Research Centre",
 	"license": "BSD-3-Clause",
 	"dependencies": {
-		"axios": "0.20.0",
-		"eslint-config-fluid": "1.4.0",
-		"fluid-grunt-lint-all": "2.0.0",
-		"jsdom": "16.4.0",
-		"json5": "2.1.3",
-		"rimraf": "3.0.2",
-		"simple-git": "2.21.0"
+		"@octokit/core": "3.5.1",
+		"axios": "0.21.1",
+		"json5": "2.2.0"
 	},
 	"devDependencies": {
-		"grunt": "1.3.0",
+		"dotenv": "10.0.0",
+		"eslint-config-fluid": "2.0.0",
+		"fluid-grunt-lint-all": "2.0.0",
```

Comment:
We should move away from the grunt-based linting tools to fluid-lint-all - this can be done in a separate pull